graph/iterator.go

```diff
@@ -153,6 +153,9 @@ type Description struct {
 	Iterators []Description  `json:",omitempty"`
 }
 
+// A curried function that can generates a new iterator based on some prior iterator.
+type ApplyMorphismFunc func(QuadStore, Iterator) Iterator
```

Comment:
s/ApplyMorphismFunc/ApplyMorphism/

Also per go vet, "// ApplyMorphism is a "...